src/pfe/portal/modules/User.js

```diff
@@ -639,54 +639,51 @@ module.exports = class User {
   /**
    * Function to write workspace settings
    */
-  async writeWorkspaceSettings(workspaceSettings) {
+  async writeWorkspaceSettings(address, namespace) {
     let retval;
     try {
       log.info(`Writing workspace settings file.`);
-      retval = await this.fw.writeWorkspaceSettings(workspaceSettings);
+      retval = await this.fw.writeWorkspaceSettings(address, namespace);
     } catch (err) {
       log.error(`Error in writeWorkspaceSettings`);
-      log.error(err);
+      throw err;
     }
     log.debug(`writeWorkspaceSettings return value: ` + JSON.stringify(retval));
     return retval;
   }
 
   /**
-   * Function to get test deployment registry
+   * Function to get test image push registry
    */
-  async testDeploymentRegistry(deploymentRegistry) {
+  async testImagePushRegistry(address, namespace) {
     let retval;
     try {
-      retval = await this.fw.testDeploymentRegistry(deploymentRegistry);
+      retval = await this.fw.testImagePushRegistry(address, namespace);
     } catch (err) {
-      log.error(`Error in testDeploymentRegistry`);
-      log.error(err);
+      log.error(`Error in testImagePushRegistry`);
```

Comment:
Same as previous comment

Reply:
Error is logged in route

```
 if (operation === undefined || operation === 'test') {
      log.debug(`Testing image push registry: ${address}/${namespace}`);
      retval = await user.testImagePushRegistry(address, namespace);
    } else if (operation === 'set') {
      log.debug(`Setting image push registry: ${address}/${namespace}`);
      retval = await user.writeWorkspaceSettings(address, namespace);
    }
    res.status(retval.statusCode).send(retval);
  } catch (error) {
    log.error(error);
    const workspaceSettings = {
      statusCode: 500,
      imagePushRegistryTest: false,
      msg: error.message
    }
    res.status(500).send(workspaceSettings);
  }
```

didnt want multiple places to have same error logged